services/spd/tlkd/tlkd_main.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2015, ARM Limited and Contributors. All rights reserved.
+ * Copyright (c) 2015-2016, ARM Limited and Contributors. All rights reserved.
```

Comment:
In that case, no I don't think so - I was assuming that this was a new patch as one of the other files had a 2017 date.